src/Bridges/lazybridgeoptimizer.jl

```diff
@@ -105,5 +105,7 @@ function supports_bridging_constraint(b::LazyBridgeOptimizer, F::Type{<:MOI.Abst
 end
 function bridge_type(b::LazyBridgeOptimizer{BT}, F::Type{<:MOI.AbstractFunction}, S::Type{<:MOI.AbstractSet}) where BT
     update_constraint!(b, F, S)
-    b.best[(F, S)]
+    get(b.best, (F, S)) do
```

Comment:
This use of the `do` syntax seems a bit unusual. IMO the following would be easier to read:
```julia
result = get(b.best, (F, S), nothing)
if result !== nothing
   return result
else
   throw(...)
end
```

Reply:
I am worried that with this construct the compiler might fail to deduce that the function never returns `nothing`. With this, the inferrability of the result is certain.
If followed the docstring of `get`:
```
get(f::Function, collection, key)

  Return the value stored for the given key, or if no mapping for the key is
  present, return f(). Use get! to also store the default value in the
  dictionary.

  This is intended to be called using do block syntax

  get(dict, key) do
      # default value calculated here
      time()
  end
```

Reply:
Indeed, `typeof(f())` is `Nothing` so it might be equivalent